backend/FwLite/MiniLcm/Validators/MiniLcmValidators.cs

```diff
@@ -4,12 +4,49 @@
 
 namespace MiniLcm.Validators;
 
-public record MiniLcmValidators(IValidator<ComplexFormType> ComplexFormTypeValidator)
+public record MiniLcmValidators(
+    IValidator<ComplexFormType> ComplexFormTypeValidator,
+    IValidator<Entry> EntryValidator,
+    IValidator<Sense> SenseValidator,
+    IValidator<ExampleSentence> ExampleSentenceValidator,
+    IValidator<WritingSystem> WritingSystemValidator,
+    IValidator<PartOfSpeech> PartOfSpeechValidator,
+    IValidator<SemanticDomain> SemanticDomainValidator)
 {
     public async Task ValidateAndThrow(ComplexFormType value)
     {
         await ComplexFormTypeValidator.ValidateAndThrowAsync(value);
     }
+
+    public async Task ValidateAndThrow(Entry value)
+    {
+        await EntryValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrow(Sense value)
+    {
+        await SenseValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrow(ExampleSentence value)
+    {
+        await ExampleSentenceValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrow(WritingSystem value)
+    {
+        await WritingSystemValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrow(PartOfSpeech value)
+    {
+        await PartOfSpeechValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrowAsync(SemanticDomain value)
```

Comment:
the name here is not consistent with the rest

Reply:
Missed that one in the rename earlier; fixed in commit 7e0e6045.